topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -110,21 +111,7 @@ But what does this quality mean?
 
 The quality for each sequence is a string of characters, one for each base of the nucleic sequence, used to characterize the probability of mis-indentification of each base. The score is encoded using the ASCII character table (with [some historical differences](https://en.wikipedia.org/wiki/FASTQ_format#Encoding)):
 
-```
- SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS...............................  Sanger
- ..........................XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX  Solexa
- ...............................IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII  Illumina 1.3+
- .................................JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ  Illumina 1.5+
- LLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLL..............................  Illumina 1.8+
- !"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefgh
- |                         |    |        |                              |
-33                        59   64       73                            104
- 0........................26...31.......40
-                          -5....0........9.............................40
-                                0........9.............................40
-                                   3.....9.............................40
- 0.2......................26...31........41
-```
+![Encoding of the quality score with ASCII characters for different Phred encoding](../../images/quality_score_encoding.png)
```

Comment:
Technically the Illumina 1.8 version of the format seems to be ASCII 33 to 93:

http://resources.qiagenbioinformatics.com/manuals/clcgenomicsworkbench/700/Quality_scores_in_Illumina_platform.html

This is also discussed in the discussion of the wikipedia entry. Though, to me it seems that values >40 don't appear with current sequencers/software. 

This is also a problem for the data format sniffer. As I remember Galaxy just takes everything as fastqsanger.

Reply:
Should we add a note for that? I am afraid it makes it even more complicated

Reply:
I agree this would probably only be confusing for people, and we can probably leave it out since this tutorial is meant as a first introduction to QC?